### ISSUE TITLE
Acceptance tests are failing in Github Actions

### ISSUE THREAD

**Issue by ozerovandrei** [hashicorp/terraform-plugin-sdk#565]
### SDK version
```
v2.0.1
```

### Debug Output

https://github.com/selectel/terraform-provider-selectel/runs/1043790743?check_suite_focus=true
https://github.com/selectel/terraform-provider-selectel/runs/1071837348?check_suite_focus=true

### Expected Behavior

Acceptance tests should run without strange errors

### Actual Behavior

I'm getting the following errors while running Acceptance tests on the provider with the v2 SDK on the `ubuntu-latest` runner:

```
=== RUN   TestAccDomainsDomainV1ImportBasic
cannot run Terraform provider tests: executable found at path /usr/local/bin/terraform is not terraform: located executable at /usr/local/bin/terraform, but output of `terraform version` was:

Your version of Terraform is out of date! The latest version
is 0.13.1. You can update by downloading from https://www.terraform.io/downloads.html
Terraform v0.13.0

FAIL	github.com/terraform-providers/terraform-provider-selectel/selectel	5.686s
```

I tried to add the [hashicorp/setup-terraform@v1]() step in the worfklow configuration, removed `TF_LOG` env variable and `-v` argument from the `go test` command and now I'm getting the following error:

```
cannot run Terraform provider tests: executable found at path /home/runner/work/_temp/839df762-2d41-4108-815e-916ae2727fec/terraform is not terraform: located executable at /home/runner/work/_temp/839df762-2d41-4108-815e-916ae2727fec/terraform, but output of `terraform version` was:
[command]/home/runner/work/_temp/839df762-2d41-4108-815e-916ae2727fec/terraform-bin version
Terraform v0.13.2

FAIL	github.com/terraform-providers/terraform-provider-selectel/selectel	2.656s
```

### Steps to Reproduce

Trying to run Acceptance tests in the Github Actions on the `ubuntu-latest` runner with the workflow similar to https://github.com/selectel/terraform-provider-selectel/blob/master/.github/workflows/acceptance-tests.yml

### References

https://github.com/selectel/terraform-provider-selectel/issues/135
https://github.com/selectel/terraform-provider-selectel/blob/master/.github/workflows/acceptance-tests.yml
https://github.com/selectel/terraform-provider-selectel/runs/1043790743?check_suite_focus=true
https://github.com/selectel/terraform-provider-selectel/runs/1071837348?check_suite_focus=true

**Comment by bflad** [comment]
Hi @ozerovandrei 👋  Drive-by comment, but does it work if you set the `hashicorp/setup-terraform` action `terraform_wrapper` input to `false`?

```yaml
- uses: hashicorp/setup-terraform@v1
  with:
    terraform_wrapper: false
```

By default the GitHub Action adds a little magic around the normal Terraform binary to make outputs available, which is generally incompatible with other tooling.

**Comment by kvrhdn** [comment]
I've had the same issue. I can confirm it was fixed by adding the following (similar to what @bflad already proposed):

```yaml
      - uses: hashicorp/setup-terraform@v1
        with:
          terraform_version: 0.13.1
          terraform_wrapper: false
```
https://github.com/kvrhdn/terraform-provider-honeycombio/blob/main/.github/workflows/ci.yaml#L23

Terraform 0.13.0 has a regression which messes up the order of `terraform version`. Ever since there is a newer release its output breaks the acceptance tests.

**Comment by kvrhdn** [comment]
This has been fixed in Terraform 0.13.1: https://github.com/hashicorp/terraform/pull/25811
So I'm not sure what the SDK can do about this now.

**Comment by ozerovandrei** [comment]
@bflad @kvrhdn thank you guys for helping me with that!

This thing works!
```
- uses: hashicorp/setup-terraform@v1
  with:
    terraform_wrapper: false
```
